dao/mongodb.rya/src/main/java/mvm/rya/mongodb/instance/MongoDetailsAdapter.java

```diff
@@ -167,7 +164,7 @@ public static RyaDetails toRyaDetails(final DBObject mongoObj) throws MalformedR
         return RyaDetails.builder()
             .setRyaInstanceName(basicObj.getString(INSTANCE_KEY))
             .setRyaVersion(basicObj.getString(VERSION_KEY))
-            .setEntityCentricIndexDetails(new EntityCentricIndexDetails(basicObj.getBoolean(ENTITY_DETAILS_KEY)))
+            .setEntityCentricIndexDetails(new EntityCentricIndexDetails(false))//entity indexing is not supported
             .setGeoIndexDetails(new GeoIndexDetails(basicObj.getBoolean(GEO_DETAILS_KEY)))
             .setPCJIndexDetails(getPCJIndexDetails(basicObj))
```

Comment:
I did it this way as a catch all placeholder for now.  it was the fastest way to get the job done.  A conversation does need having